pkg/ddl/executor.go

```diff
@@ -4070,6 +4070,9 @@ var systemTables = map[string]struct{}{
 }
 
 func isUndroppableTable(schema, table string) bool {
+	if schema == "workload_schema" {
```

Comment:
Maybe we should use const in `repository` package. Removing depedency is great, but it will also be convenient when we want to know all occurences of schema table name by searching workloadSchemaCIStr.